vhdbuilder/packer/pre-install-dependencies.sh

```diff
@@ -85,9 +85,11 @@ if isMarinerOrAzureLinux "$OS"; then
     installFIPS
   fi
 else
-  # Enable ESM for 18.04 and FIPS only
+  # Enable ESM only for 18.04, 20.04, and FIPS
   if [[ "${UBUNTU_RELEASE}" == "18.04" ]] || [[ "${UBUNTU_RELEASE}" == "20.04" ]] || [[ "${ENABLE_FIPS,,}" == "true" ]]; then
```

Comment:
We discussed and we should enable PRO for all Ubuntu releases,it allows picking up security patches from canonical instead of relying on the community repo patches. 

Canonical said it was fine, also, we have a Ticket to the that as well created by @cameronmeissner  :D

Reply:
we previously did this but decided to revert since we can't publish pro-enabled images to community image galleries per contract with Canonical, unless you're saying this is what they've now agreed to?

at the moment 1804/2004/fips SKUs are pro-enabled, and NOT published to community image galleries

Reply:
aint we disabling pro at the end ?

Also, there was this ticket, should we just close it ? or at least explain that we don't need since Pro would be enabled.  ?

I'm a bit confused there, what is the issue in the end, it's the fact that we would be publishing non pro image with packages coming from a pro repo ?

Reply:
lets discuss this offline, we don't need it for now